### PR TITLE
Fix/activesetup

### PR DESCRIPTION
**Pull request by lstipakov**

### CODE REVIEW

File: windows-msi/script/ActiveSetupCA.js [hunk 1]
```diff
@@ -74,11 +76,15 @@ function PublishActiveSetup()
...
             wsh = new ActiveXObject("WScript.Shell"),
-            regPath = "HKLM\\Software\\Microsoft\\Active Setup\\Installed Components\\" + data[1] + "\\";
+            regPath = "HKLM\\Software\\Microsoft\\Active Setup\\Installed Components\\" + data[1] + "\\",
+            hkcu_run = "HKCU\\Software\\Microsoft\\Windows\\CurrentVersion\\Run";
 
         switch (data[0].toLowerCase()) {
             case "install":
-                if (data.length >= 4) {
+                if (data.length >= 6) {
+                    var runOnLogon = data[4];
```
[the remark this answers is not in the document]
Comment (lstipakov) [reply windows-msi/script/ActiveSetupCA.js]: Oops. Fixed!

**Comment by lstipakov** [comment]
@rozmansi @selvanair is there anything which is missing here, apart from disabling repair/modify? We want to make beta2 tomorrow, would be nice to get this fix into there.

**Comment by selvanair** [comment]
This looks okay to me.

As for beta2, the installer allowing any user to delete and modify the installed files is a critical issue identified in beta1. Can beta2 wait a bit longer if we can get that fixed? Unless we plan a beta3.

I have no idea how to disable or restrict repair/modify. @rozmansi ?

**Comment by mattock** [comment]
Doing a full release is a rather heavy process for me. From that perspective I'd _prefer_ not to release beta2 until all the important fixes are in.

**Comment by rozmansi** [comment]
I have disabled Modify/Change in https://github.com/OpenVPN/openvpn-build/pull/185

**Comment by lstipakov** [comment]
> I have disabled Modify/Change in #185

That doesn't unfortunately prevent `msiexec /fu`. Also repair/reinstall radiobuttons are seen when double-clicking on MSI file.

**Comment by mattock** [comment]
I take it this is ready to go in?

**Comment by rozmansi** [comment]
> 
> 
> I take it this is ready to go in?

LGTM

**Comment by rozmansi** [comment]
The 15627f34e02a1b79725dcf27035db66ac57ff127 is a duplicate of ce1a9f21a12b5ff672cf9448e91851c57801108b just merged as a part of #185.

**Comment by lstipakov** [comment]
Rebased on top of master,

**Comment by selvanair** [comment]
With the "best effort fix" for the MSI already in, let's merge this before beta2.
ACK from me.

**Pull request merged**